menu.lua

```diff
@@ -2329,6 +2329,12 @@ function menu_keypressed(key, unicode)
 			end
 			gamestate = "menu"
 			saveconfig()
+
+			continueavailable = false
+			if love.filesystem.getInfo(mappackfolder:gsub("mappacks", "saves") .. "/" .. mappack .. ".suspend") then
+				continueavailable = true
+			end
```

Comment:
I know there's repetitive code all over Mari0 but, definitely could make a function for this haha